@here/harp-mapview/lib/MapView.ts

```diff
@@ -749,7 +750,7 @@ export class MapView extends THREE.EventDispatcher {
     private m_enablePolarDataSource: boolean = true;
 
     // gestures
-    private readonly m_raycaster = new THREE.Raycaster();
+    private readonly m_raycaster = new PickingRaycaster(this);
```

Comment:
I would prefer to get rid of this `PickingRaycaster` or at least not keep the whole `MapView` reference b/c as we know from history people will do bad things if the have access to the whole `MapView`.
As far as I can see, the `PickingRaycaster` is only used in `MapViewPoints` and there we use the camera and the canvas size. Would be cleaner to add these properties explicitly and we would also avoid to query the DOM for every pick event.
Long story short: I would prefer if you do the changes now but if you don't have time, please create a ticket.

Reply:
I will fix now, seems to be a small change.